.github/workflows/release.yaml

```diff
@@ -71,6 +90,166 @@ jobs:
+          git push origin main
+          git push origin $tagName
+
+  kotlin-release-to-tbd-artifactory-and-maven-central:
+    name: Release Kotlin to Maven Central
+    needs: git-tag
+    runs-on: ubuntu-latest
+    steps:
+
+      # Check out the tag we created above
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+          ref: ${{ needs.git-tag.outputs.RELEASE_TAG }} # Check out the tag we created above
+
+      # Set up JDK
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: "adopt"
+          java-version: "11"
+
+      # Cache Maven repo
+      - uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      # Package up the native binaries
+      #TODO Centralize this block as we re-use it via copy/paste right now
+      - name: Download MacOS aarch64 Native Library
+        uses: actions/download-artifact@v4.1.7
+        with:
+          name: aarch64-apple-darwin-dylib
+          path: bound/kt/src/main/resources/
+      - name: Download MacOS x86_64 Native Library
+        uses: actions/download-artifact@v4.1.7
+        with:
+          name: x86_64-apple-darwin-dylib
+          path: bound/kt/src/main/resources/
+      - name: Download Linux x86_64 GNU Native Library
+        uses: actions/download-artifact@v4.1.7
+        with:
+          name: x86_64-unknown-linux-gnu-so
+          path: bound/kt/src/main/resources/
+      - name: Download Linux x86_64 MUSL Native Library
+        uses: actions/download-artifact@v4.1.7
+        with:
+          name: x86_64-unknown-linux-musl-so
+          path: bound/kt/src/main/resources/
+      - name: Download Windows x86_64 MSVC Native Library
+        uses: actions/download-artifact@v4.1.7
+        with:
+          name: x86_64-pc-windows-msvc-dll
+          path: bound/kt/src/main/resources/
+
+      - name: Deploy Release Version of Kotlin Project to Maven Central
+        run: |
+          # cd into the Kotlin project
+          cd bound/kt/
+          
+          # Publish to Sonatype
+          # Precondition check: Only attempt to publish artifact if we have credentials
+          if [ -n "${{ secrets.SONATYPE_PASSWORD }}" ]; then
+            # Maven deploy lifecycle will build, run tests, verify, sign, and deploy
+            mvn deploy --batch-mode \
+              --settings .maven_settings.xml \
+              -P ossrh,sign-artifacts
+          else
+              echo "Error: No credentials"
+              exit 1
+          fi
+        env:
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SIGN_KEY_PASS: ${{ secrets.GPG_SECRET_PASSPHRASE }}
+          SIGN_KEY: ${{ secrets.GPG_SECRET_KEY }}
+
```

Comment:
I think you also need to update this release to include similar versioning strategy that you added to dokka in tbdex-rs below: https://github.com/TBD54566975/tbdex-rs/pull/120/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R204-R206

Reply:
yup good call